app/schemas/application_schema.py

```diff
@@ -0,0 +1,23 @@
+from marshmallow import validate, Schema, fields
+from .period_schema import PeriodSchema
+
+
+class ApplicationBodySchema(Schema):
```

Comment:
move to separate file, one class per file.

Reply:
> move to separate file, one class per file.

done!